Updated Router settings, save and load properties from properties file, refactored five axis programs to subdirectory, added CAD models

KentoHyono opened the pull request:
1. Updated router dimension according to the specification in the following image: 
&nbsp;<img width="755" alt="routerDimensions" src="https://github.com/user-attachments/assets/bb571fdd-199e-452f-9fe9-8af9a3c3be30">

2. Updated the colour of the cutting bit and nose of the router to red.
3. Router and bit properties value are taken and saved to the properties file.
   - Example.java,  FiveAxisConfig.java, and ThreePlusTwoPrompt.java use the same properties file.
4. Made program to store properties files under the config folder instead of the root directory.
5. Updated .gitignore file.
6. Refactored five axis programs to subdirectory.
7. Added complex CAD model for symposium.

Hanson2258 committed:
Move all Five Axis programs to FiveAxis subdirectory

Hanson2258 committed:
Upload Complex CAD Model

CAD Model to show all capabilities of a 3+2 Axis CNC Router.

Hanson2258 committed:
Merge branch 'main' into Bit-By-Bit-Capstone-Team

KentoHyono committed:
Made form save every data in cam.properties file so it can be used in Exmaple.java

KentoHyono committed:
Organizing by tab done (could change tab design so it fits in one page

KentoHyono committed:
Finished Bit UI

KentoHyono committed:
Router and bit dimensions are taken from properties file now

KentoHyono committed:
Completed feature: take and save data from the cutting config and apply it in the Examples.java. FiveAxisConfig.java and ThreePlusTwoPrompt.java use same properties file for storing data

KentoHyono committed:
Updated .gitignore

KentoHyono committed:
Corrected router dimensions

KentoHyono committed:
cam.properties and <bit name>.properties files are saved and loaded to/from config directory

KentoHyono committed:
Addded comment regardning his feedback

KentoHyono committed:
Commit properties file for test

KentoHyono committed:
Merge branch 'main' into Bit-By-Bit-Capstone-Team

Hanson2258 commented:
Examples.java and ThreePlusTwoPrompt.java are not in the FiveAxis subdirectory. Need to be refactored.

Hanson2258 committed:
Moved Examples.java and ThreePlusTwoPrompt.java into Five Axis subdirectory

Hanson2258 committed:
Moved CAD Model to Sample_Models directory

- Moved the All Capabilities Model to Sample_Model directory
- Deleted Model directory
- Renamed files and subdirectory name to be a mix of Pascal and Snake case (following the directory's naming convention)
